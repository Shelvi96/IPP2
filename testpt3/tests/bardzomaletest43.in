NEW ID

678>0:40:2;212
328;900>49
3952>;105
2160392408>65;353010
417:1:45>978770
189983>7:;
2034993;:1>79125019
266;2122;5>2:0:79
6;22633097>:534
77519466:4>034
:7802;7202>;;;3;:24;
069328>64
8;>0587
11801>74::
15473;2>7878
:5841>4684
1>69
81>1668
@;88644590137998
@8982;2:0
@;:7088;4
@:117:5851
@396;:65983:1689689
@5586:384325
@96:0445
@5046607514362340:0816
@13;33878:0027;664;;;9521
@06650;:431;;8011070;56
@5962288006:561
