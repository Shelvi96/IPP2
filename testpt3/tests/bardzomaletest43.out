31
0
0
0
4160
0
0
11600000
199087521552
127000010
1
